src/code/views/simulation-run-panel-view.coffee

```diff
@@ -21,6 +23,10 @@ module.exports = React.createClass
       SimulationStore.actions.collapseSimulationPanel()
     else
       SimulationStore.actions.expandSimulationPanel()
+      # -- TBD: There was discussion about automatically showing
+      # -- MiniGraphs when this panel is opened  …  NP 2018-01
+      # if ! @state.showingMinigraphs
+      #   AppSettingsStore.actions.showMinigraphs true
```

Comment:
This comment is no longer applicable, because this feature is implemented elsewhere, right?